tests/models/layoutlmv3/test_modeling_tf_layoutlmv3.py

```diff
@@ -278,13 +278,7 @@ class TFLayoutLMv3ModelTest(TFModelTesterMixin, PipelineTesterMixin, unittest.Te
         else ()
     )
     pipeline_model_mapping = (
-        {
-            "feature-extraction": TFLayoutLMv3Model,
-            "question-answering": TFLayoutLMv3ForQuestionAnswering,
-            "text-classification": TFLayoutLMv3ForSequenceClassification,
-            "token-classification": TFLayoutLMv3ForTokenClassification,
-            "zero-shot": TFLayoutLMv3ForSequenceClassification,
-        }
+        {"document-question-answering": TFLayoutLMv3ForQuestionAnswering, "feature-extraction": TFLayoutLMv3Model}
```

Comment:
Those were added by users or @Rocketknight1 maybe. In any case, `pipeline_model_mapping` should be added/modified with the script `add_pipeline_model_mapping_to_test.py`.

Also, the whole pipeline test for `(TF)LayoutLMv3ModelTest` is skipped, and need a fix to re-enable it.